ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/update_available/util/PlayStoreUtil.kt

```diff
@@ -0,0 +1,46 @@
+@file:Suppress("PackageNaming")
+package co.electriccoin.zcash.ui.screen.update_available.util
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+
+object PlayStoreUtil {
+
+    const val PLAY_STORE_APP_URI = "market://details?id="
+
+    const val FLAGS = Intent.FLAG_ACTIVITY_NO_HISTORY or
+        Intent.FLAG_ACTIVITY_NEW_TASK or
+        Intent.FLAG_ACTIVITY_NEW_DOCUMENT or
```

Comment:
Why `Intent.FLAG_ACTIVITY_NEW_DOCUMENT`?

Reply:
We do not need this one, that's correct. FLAG_ACTIVITY_NEW_TASK is necessary and FLAG_ACTIVITY_NEW_DOCUMENT can be omitted. Thank you for pointing it out.